### ISSUE TITLE
How to predict new examples? 

### ISSUE THREAD

**Issue by ptgamr** [VladislavPrh/MTL#1]
As I understand from your examples.py file, you're generate predictions for img in the test set. However it come through the same preprocess stage with bounding box defined in the CSV.

However, with new examples, if we don't have bounding box defined, how can we crop & resize the image?

Many thanks for your help.

**Comment by VladislavPrh** [comment]
We should predict bounding box first, therefore we need to train separate regressor to do it. We can do it for example by creating one more 'head' in Neural Network or train another model.

**Comment by ptgamr** [comment]
Hi there,

My original image is not too big, it's already be cropped to the size that are quite small already. I thought with the method in this paper, we can still regconize from some raw input  images like this

![test6](https://user-images.githubusercontent.com/1619740/38719615-078fb4a8-3f47-11e8-8f4b-7e3701620af1.png)

I tried to fit the raw image to the network, but the result didn't look too promising. 

Seems like it only work if we can feed the images that are cropped closely to the number.
